src/modules/peek/Peek.FilePreviewer/FilePreview.xaml

```diff
@@ -18,6 +18,11 @@
             VerticalAlignment="Center"
             IsActive="{x:Bind MatchPreviewState(Previewer.State, previewers:PreviewState.Loading), Mode=OneWay}" />
 
+        <controls:ShellPreviewHandlerControl
```

Comment:
Is it always visible? Should it have a toggle for visibility?

Reply:
It has its own way of handling visibility so resize events would work, but I could add another visibility property for consistency.